common/app/common/ModelOrResult.scala

```diff
@@ -77,16 +77,18 @@ object InternalRedirect extends implicits.Requests with GuLogging {
       .orElse(response.tag.map(t => internalRedirect("facia", t.id)))
       .orElse(response.section.map(s => internalRedirect("facia", s.id)))
 
+  private def pathFromWebUrl(webUrl: String): String =
+    webUrl.replaceFirst("^https?://www.theguardian.com/", "")
```

Comment:
using the same idea as in https://github.com/guardian/frontend/pull/25627/files#diff-c0d95c7fbd0804c5ba49a2dbb3c06d44f3c6a1e75988fea29eccb6aecf88b047R23